index.html

```diff
@@ -58,5 +58,224 @@ <h3>LET'S CONNECT</h3>
       </div>
     </section>
   </div>
+
+
+      <!--Work section start-->
+      <section id="work-section">
+        <div class="_container">
+          <div class="work-content">
+            <!--card section 1-->
+              <div class="card card1">
+                <img src="Images/p1.png" alt="picture 1">
+                <div class="card--content">
+                  <h2 class="title">Tonic</h2>
+                  <ul class="items">
+                    <li class="item"><span>CANOPY</span></li>
+                    <li class="dot-item"><span></span></li>
+                    <li class="item"><span>Back End Dev</span></li>
+                    <li class="dot-item"><span></span></li>
+                    <li class="item"><span>2015</span></li>
+                  </ul>
+                  <p class="desc">
+                    A daily selection of privately personalized reads; no accounts
+                    or sign-ups required.
+                  </p>
+                  <div class="lang-elment">
+                    <ul class="lang">
+                      <li class="lan"><span>html</span></li>
+                      <li class="lan"><span>css</span></li>
+                      <li class="lan"><span>javaScript</span></li>
+                    </ul>
+                  </div>
+                  <div class="button-f">
+                    <button class="btn" type="button">See Project</button>
+                  </div>
+                </div>
+              </div>
+            <!--End card section 1-->
+
+            <!--card section 1-->
+            <div class="card card1">
+              <img src="Images/p2.png" alt="picture 1">
```

Comment:
- [ ] Your pictures are not being displayed. Please, fix that.
![image](https://user-images.githubusercontent.com/61919617/170836990-7d177ec3-3455-4f08-affe-17a51b763150.png)

Reply:
Can you please check it again, because I can see it in my browser.